client/components/section-header/docs/example.jsx

```diff
@@ -23,16 +23,26 @@ var Cards = React.createClass( {
 				</h2>
 
 				<SectionHeader label="Team" count={ 10 }>
-					<SectionHeaderButton>Manage</SectionHeaderButton>
-					<SectionHeaderButton onClick={ function() {
-						alert( 'Clicked add button' );
-					} }>
-						Add
-					</SectionHeaderButton>
+					<Button
+						compact
+						primary>
+						{ this.translate( 'Primary Action' ) }
```

Comment:
I could be wrong, but I don't think we need to translate the strings in examples. If we do keep the translation, we should probably at least provide some context. Maybe something like,  `this.translate( 'Primary Action', { context: 'Button display in documentation' } )`?

cc @deBhal who might know whether we need to translate these strings.

__Note:__ There's also another translated string in this file on line `33`.